AwaMelvine/fast-food-fast
# Fix issue with items image upload 164998951

#### What does this PR do?
Fixes issue with image upload on create items form

#### Description of Task to be completed?
Make available the rootUrl variable in the create_items file on the client application.
#### How should this be manually tested?
- Log in as admin and navigate to the create food items page.
- Create a new item, selecting an image for the item, then click on the create item button
- Now check the items page and check that the image is displayed along the newly created food item

#### What are the relevant pivotal tracker stories?
[#164998951] Fix issue with image upload on food items form
